sonaranalyzer-dotnet/rspec/vbnet/S2737_vb.net.html

```diff
@@ -0,0 +1,27 @@
+<p>A <code>Catch</code> clause that only rethrows the caught exception has the same effect as omitting the <code>Catch</code> altogether and letting
+it bubble up automatically, but with more code and the additional detriment of leaving maintainers scratching their heads. </p>
+<p>Such clauses should either be eliminated or populated with the appropriate logic.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+Dim s As String = ""
+Try
+    s = File.ReadAllText(fileName)
+Catch e As Exception
+    Throw
+End Try
+</pre>
+<h2>Compliant Solution</h2>
+<pre>
+Dim s As String = ""
+Try
+    s = File.ReadAllText(fileName)
+Catch e As Exception
+    logger.LogError(e)
+    Throw
+End Try
+</pre>
+<p>or</p>
+<pre>
+Dim s As String = File.ReadAllText(fileName)
+</pre>
```

Comment:
I am not sure if this is actually an exception from the rule. In my understanding exceptions are real issues we ignore because they are widely used. The "exceptional" case in this implementation is something that will change the application logic if removed...

Reply:
Anyway, I added the exception in RSPEC and updated the metadata